thrift/src/main/thrift/atoms/explainer.thrift

```diff
@@ -2,16 +2,59 @@ namespace * contentatom.explainer
 namespace java com.gu.contentatom.thrift.atom.explainer
 #@namespace scala com.gu.contentatom.thrift.atom.explainer
 
+include "../shared.thrift"
+
 enum DisplayType {
   FLAT = 0,
   EXPANDABLE = 1,
   CAROUSEL = 2
 }
 
 struct ExplainerAtom {
+  1: optional string label
   /* the unique ID will be stored in the `atom` data*/
   2: required string title
+  /* hopefully we can get rid of this one later: */
   3: required string body
   4: required DisplayType displayType
   5: optional list<string> tags
+  6: optional ExplainerData explainerBody
```

Comment:
To make your lives much easier and circumvent having to write specific code to serialise/deserialise this structure, I would advise that you store each different type on the model as its own optional field. Much like we do for reviews. It feels less 'beautiful' but we've found it to be significantly easier in practice.